docs/docs/reaching-out-to-user.mdx

```diff
@@ -0,0 +1,459 @@
+---
+id: reaching-out-to-user
+sidebar_label: Reaching Out to the User
+title: Reaching Out to the User
+---
+
+Sometimes you want your assistant to reach out to the user without the user's prompting.
+For example, you might want the assistant to send a message when the user opens the chat window,
+or you might want to prompt the user if they haven't sent a message for a while.
+This page is a guide to enabling your assistant to reach out to the user proactively.
+
+
+## Reaching out first
+
+To have your assistant reach out to the user at the beginning of a conversation,
+configure your [messaging or voice channel](messaging-and-voice-channels.mdx) to send a payload to the bot that
+will trigger the appropriate response. For example, in Sara, the Rasa bot, when a
+user opens the chat window, the chat widget sends the payload `/greet`, so that
+Sara greets the user with a welcome message. How you send the initial payload
+will depend on the channel(s) you are using.
+
+On the bot's side,
+you might want to reuse an existing intent and response, or have a specific
+intent and response for the start of conversations. If you want a specific welcome
+message that the user won't also see by e.g. greeting the bot, you need
+to add the intent and response(s) to your domain, and add a rule
+for what the bot should do with the initial payload.
+
+### Data
+
+Using the following example training data, your assistant
+will send the `utter_welcome` response when the `chat_opened` intent is predicted:
+
+```rasa-yaml
+intents:
+- chat_opened
+
+responses:
+  utter_welcome:
+  - text: Hi there! What can I help you with today?
+
+rules:
+  - rule: welcome user
+    steps:
+      - intent: chat_opened
+      - action: utter_welcome
+```
+
+Since there's no training data for the intent `chat_opened`, the intent can only
+be triggered explicitly by sending `/chat_opened`, which is what your channel
+should send to the bot.
+
+### Try it Out
+
+If you don't yet have a channel set up for your assistant, you can use [rasa-webchat](https://github.com/botfront/rasa-webchat),
+a [websocket channel](./connectors/your-own-website.mdx)
+and configure the [`initPayload`](https://github.com/botfront/rasa-webchat#parameters) to send the message `/chat_opened`.
+After retraining your model, start the rasa server:
+
+```shell
+rasa run --enable-api --debug
+```
+
+In the `rasa-webchat` widget, you should now be greeted with the message, "Hi there! What can I help you with today?"
```

Comment:
we should maybe take out the whole "Try it out" section? Or say you can talk to sara to try it out?